pkg/encoding/fix_test.go

```diff
@@ -0,0 +1,184 @@
+/*
+Copyright 2017 The Kedge Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package encoding
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/kedgeproject/kedge/pkg/spec"
+
+	api_v1 "k8s.io/client-go/pkg/api/v1"
+)
+
+func TestFixServices(t *testing.T) {
+	failingTest := []spec.ServiceSpecMod{
```

Comment:
How does it become difficult to understand, there is a field right there next to the test which says if the test is expected to pass or fail.
Also, if we separate passing and failing tests, we are sorting by test behavior not by the functionality that is being tested. One behavior will have passing and failing tests defined separately and will be very difficult to track the tests for a given behavior.

If we absolutely have to make things concise, we can have different []struct for different behavior.